### PR TITLE
#84 generating gitignore

### PR DESCRIPTION
**Pull request by maciej-majchrzak-wttech**

### CODE REVIEW

File: project-init.sh [hunk 1]
```diff
@@ -18,6 +18,8 @@ echo ""
 chmod +x "${AEM_WRAPPER}" "${TASK_WRAPPER}"
 sh ${AEM_WRAPPER} version
 
+echo "# AEM Compose\naem/home\ndispatcher/target" >> .gitignore
```
Comment (krystian-panek-wttech) [inline project-init.sh:21]: already initializiation logic is done in `aem project init` so I am not sure if introducing second place is a good way to do;

It's for me not clear what then should be done here and what in `aem project init` cmd

Comment (maciej-majchrzak-wttech) [reply project-init.sh:21]: 1. `project-init.sh` is called once, by a person setting up the project for the first time
2. `aem project init` is called while setting up the project but it's also called by all team members when they setup their machines (or whatever other environment)
Therefore, `.gitignore` should be generated in `project-init.sh`.

However, I'm not convinced about the `aem project init` command. Currently, it does two things and it's confusing: it sets up initial structure of the project (the one kept in git repo), but also it must be called by all team members to setup their local envs. The latter is not really "project init", it's more like "env init", and ideally it should not be needed to be called by devs - it should simply be part of `aem instance create`. If that's the case, then we could move `.gitignore` generation to under `aem project init`. Does it make sense to you?

Comment (krystian-panek-wttech) [reply project-init.sh:21]: my idea is that `aem project init` works despite the current VCS status, and it's self-sufficient to determine what to do in whatever state. to always ensure a workable state

Comment (krystian-panek-wttech) [reply project-init.sh:21]: > we could move .gitignore generation to under aem project init. Does it make sense to you?

yes

Comment (krystian-panek-wttech) [reply project-init.sh:21]: > Currently, it does two things and it's confusing: it sets up initial structure of the project (the one kept in git repo), but also it must be called by all team members to setup their local envs. 

that's intentional to not confuse the users to distinguish the states they have

Comment (krystian-panek-wttech) [reply project-init.sh:21]: probably `aem project init` should be just `aem init`

**Comment by krystian-panek-wttech** [comment]
done differently; in Go / `aem init` cmd

https://github.com/wttech/aemc/blob/main/pkg/project/project.go#L151

**Pull request closed without merging**